### DISCUSSION TITLE
Add `color-scheme` property

### DISCUSSION THREAD

**Discussion by dungsil** [RyanClementsHax/tailwindcss-themer#103] [Ideas]
Hello.
Thank you for creating such a great open source. 

Some browsers render colors based on the [`color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme) property of the browser's elements (e.g., scrollbars in Microsoft Edge).

<details>
<summary> Example GIF </summary>

![](https://github.com/RyanClementsHax/tailwindcss-themer/assets/39783731/27b2412c-36ce-4b30-92de-6b408eb48193)

</details>

I wrote a new plugin to handle this, but it would be nice if it could be applied together in `tailwindcss-themer`.
```js
// ...
addBase({
  '.light': { colorScheme: 'only light' },
  '.dark': { colorScheme: 'only dark' },
})
// ...
```

In my opinion, why not add the following options to the theme?
```js
themer({
    themes: [
        {
        	name: 'dark',
        	isDark: true,
        	extend: {}
        },
        // or 
        {
        	name: 'dark',
        	scheme: 'only dark',
        	extend: {}
        }
    ]
})
```

**Comment by RyanClementsHax** [comment]
Thanks for the suggestion. Out of curiosity, what was your implementation? Do you have a link?

**Reply by dungsil** [reply]
Hello.  The options below are examples. The only thing I added was the `addBase(...)` part above.

**Reply by RyanClementsHax** [reply]
Ah OK I see. I'll look into this further but this seems like something I'd like to support.

**Comment by zobweyt** [comment]
This plugin is great! The only feature I'm also missing is support for the `color-scheme` property. It would be nice to have that included.
